lib/product.rb

```diff
@@ -0,0 +1,34 @@
+class Product
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :brand, String
+  property :product_name, String
+  property :category, String
```

Comment:
I would go for a have n relationship between Product and Category. What are the possible pitfalls of not having that?